janus_server/src/aggregator.rs

```diff
@@ -2149,16 +2149,35 @@ fn aggregator_filter<C: Clock>(
         aggregate_share_routing,
         aggregate_share_responding,
         warp::cors().build(),
-        response_time_recorder,
+        response_time_recorder.clone(),
         "aggregate_share",
     );
 
+    let health_check_routing = warp::path("_health_check");
```

Comment:
Maybe instead of prefixing routes like this with `_`, we should try to namespace our routes to avoid future collisions. So all the HTTP routes specified by DAP could be under `/dap/` (`/dap/hpke_config`, `/dap/upload`, etc.), and then we could call our health check route whatever we want and never have to worry about colliding with new routes defined by DAP.

Reply:
There's a convention in Kubernetes/Docker for using `/healthz` for this. (https://stackoverflow.com/questions/43380939/where-does-the-convention-of-using-healthz-for-application-health-checks-come-f)